deploy/crds/infra.watch_servicetelemetrys_crd.yaml

```diff
@@ -175,11 +175,15 @@ spec:
                                     type: string
                                 type: object
                             type: object
-                          endpoint_cert_duration:
+                          endpointCertDuration:
                             description: The requested 'duration' (i.e. lifetime) of the ElasticSearch endpoint Certificate.
+                              Minimum accepted duration is 1 hour. Value must be in units accepted by Go time.ParseDuration https://golang.org/pkg/time/#ParseDuration
+                            pattern: ^((([0-9]+)h)?(([0-9]+)m)?(([0-9]+)s)?(([0-9]+)ms)?)$
                             type: string
-                          ca_cert_duration:
+                          caCertDuration:
                             description: The requested 'duration' (i.e. lifetime) of the ElasticSearch CA Certificate.
+                              Minimum accepted duration is 1 hour. Value must be in units accepted by Go time.ParseDuration https://golang.org/pkg/time/#ParseDuration
+                            pattern: ^((([0-9]+)h)?(([0-9]+)m)?(([0-9]+)s)?(([0-9]+)ms)?)$
```

Comment:
Looking at this some more, I feel like because we're adding 2 parameters here, that we should actually nest them like we do for the `storage` parameters. Perhaps add these under a top level `security` object. This would help in the future if we need to add more items (we probably won't of course), and gives a more natural setup of these parameters IMO.

Reply:
`security` or `certificates` ? Might be some other items that make more sense.

Reply:
+1 fixed that in the latest PR